_bibliography/pint.bib

```diff
@@ -2585,15 +2688,57 @@ @article{Lederman2016
         url = {http://dx.doi.org/10.1007/s40314-016-0319-7}
 }
 
-@unpublished{NielsenHesthaven2016,
-	author = {Nielsen, Allan Svejstrup and Hesthaven, Jan S.},
-	title = {Fault Tolerance in the Parareal Method},
-	year = {2016},
-	url = {http://infoscience.epfl.ch/record/216982},
-	howpublished = {EPFL-CONF-216982},
+@inproceedings{MatsuokaEtAl2016,
+	abstract = {{Slowdown and inevitable end in exponential scaling of processor performance, the end of the so-called "Moore's Law" is predicted to occur around 2025--2030 timeframe. Because CMOS semiconductor voltage is also approaching its limits, this means that logic transistor power will become constant, and as a result, the system FLOPS will cease to improve, resulting in serious consequences for IT in general, especially supercomputing. Existing attempts to overcome the end of Moore's law are rather limited in their future outlook or applicability. We claim that data-oriented parameters, such as bandwidth and capacity, or BYTES, are the new parameters that will allow continued performance gains for periods even after computing performance or FLOPS ceases to improve, due to continued advances in storage device technologies and optics, and manufacturing technologies including 3-D packaging. Such transition from FLOPS to BYTES will lead to disruptive changes in the overall systems from applications, algorithms, software to architecture, as to what parameter to optimize for, in order to achieve continued performance growth over time. We are launching a new set of research efforts to investigate and devise new technologies to enable such disruptive changes from FLOPS to BYTES in the Post-Moore era, focusing on HPC, where there is extreme sensitivity to performance, and expect the results to disseminate to the rest of IT.}},
+         author = {Matsuoka, Satoshi and Amano, Hideharu and Nakajima, Kengo and Inoue, Koji and Kudoh, Tomohiro and Maruyama, Naoya and Taura, Kenjiro and Iwashita, Takeshi and Katagiri, Takahiro and Hanawa, Toshihiro and Endo, Toshio},
+         title = {From FLOPS to BYTES: Disruptive Change in High-performance Computing Towards the Post-moore Era},
+         booktitle = {Proceedings of the ACM International Conference on Computing Frontiers},
+         series = {CF '16},
+         year = {2016},
+         location = {Como, Italy},
+         pages = {274--281},
+         numpages = {8},
+         url = {http://dx.doi.org/10.1145/2903150.2906830},
+         doi = {10.1145/2903150.2906830},
+         publisher = {ACM},
+         address = {New York, NY, USA},
+ }
```

Comment:
It takes about PINT at a few points in the paper: ` In order
to keep scalability on future supercomputer systems with
large number of nodes with hierarchical network, parallelization
in time direction for time-dependent problems is essential.`

Like the Exascale Maths report, it is might be a good paper to cite to support the importance of PINT in future HPC.